### PR TITLE
Fixing broken tests

### PR DESCRIPTION
**Pull request by ck99**

### CODE REVIEW

**Comment by bwoebi** [comment]
Thanks a lot, that's exactly what I've planned to do today :-D
Merging as soon as travis build finishes :-)

**Comment by ck99** [comment]
;)

nice!

File: lib/Websocket/Message.php [hunk 1]
```diff
@@ -2,11 +2,13 @@
...
 namespace Aerys\Websocket;
 
+use Amp\ByteStream\IteratorStream;
+
 class Message extends \Amp\ByteStream\Message {
     private $binary = false;
 
     public function __construct(\Amp\Iterator $iterator, bool $binary = false) {
-        parent::__construct($iterator);
+        parent::__construct(new IteratorStream($iterator));
```
Comment (kelunik) [inline lib/Websocket/Message.php]: I guess `Message` should just accept an `InputStream` and the `new IteratorStream` should be done when constructing the object?

Comment (bwoebi) [inline lib/Websocket/Message.php]: [That's the type of issue we get by having separate Iterator and InputStream…]
Also, I don't care at that level. The message is only instantiated within Rfc6455Gateway and meh.

**Pull request merged**